bfdd/bfd_packet.c

```diff
@@ -878,7 +878,7 @@ void bfd_recv_cb(struct event *t)
 	/*
 	 * We may have a situation where received packet is on wrong vrf
 	 */
-	if (bfd && bfd->vrf && bfd->vrf != bvrf->vrf) {
+	if (bfd && bfd->vrf && bfd->vrf->vrf_id != vrfid) {
```

Comment:
with udp_l3mdev_accept=1 why would we even want any type of log message here?  We should be creating only 1 socket for listening ( or if we create more sockets per vrf then the right thing should be done implicitly ).  As that this implies that the wrong control methodology was implemented and we are sending messages that should not even be.

Reply:
So, you mean this vrf check here should be removed?